clients/web/src/templates/body/plugins.pug

```diff
@@ -1,14 +1,10 @@
 .g-body-title Plugins
-  .g-rebuild-container
+  .g-rebuild-container(class=!cherrypyServer ? 'hide' : '')
```

Comment:
Unless this `!` operator is intentionally used to cast to boolean, we should invert this condition for readability.